src/transport/grpc/mod.rs

```diff
@@ -965,7 +963,7 @@ mod test {
                 if data_event.interface == "org.astarte-platform.rust.examples.individual-properties.DeviceProperties"
                 && data_event.path == "/1/name"
                 && matches!(data_event.data, Value::Individual(AstarteType::String(v)) if v == STRING_VALUE),
-            ServerReceivedRequest::Detach(d) if d.uuid == ID.to_string()
+            ServerReceivedRequest::Detach(a) if a == astarte_message_hub_proto::pbjson_types::Empty{}
```

Comment:
Remove this if, or move it to the match. It's already ensured by the type signature

Reply:
Same in the other locations